src/glyph.js

```diff
@@ -61,7 +61,7 @@ Glyph.prototype.bindConstructorValues = function(options) {
 
     // These three values cannot be deferred for memory optimization:
     this.name = options.name || null;
-    this.unicode = options.unicode || undefined;
+    this.unicode = options.unicode;
```

Comment:
I would suggest to explicitly set the unicode value to `undefined` if the name is `.notdef`, so that any code copied from the README prior to this fix will still result in a correct glyph. And while we're at it, we could force 0 for the `.null` name as well and maybe even prevent any other glyph from getting assigned a unicode value of 0.

Reply:
@Templarian Do you have any objections?

Reply:
@Connum That should always be the case, so that would be a good idea.